### DISCUSSION TITLE
Ender 3 S1 "Plus" version

### DISCUSSION THREAD

**Discussion by TheShaddii** [mriscoc/Ender3V2S1#239] [General]
So there is a new Ender 3 S1 Plus... As far as the electronics are concerned, I believe this is identical to the S1 except that the build volume is 300x300x300...I would be happy to open up my machine and check any details. I would really really like a copy of this firmware for the S1 Plus... After using this firmware I am completely disenchanted with the stock firmware from Creality. I believe the professional firmware is superior in every way and desperately wish to have the experience on my S1 Plus.

I will admit I have never compiled firmware before and I do not know what is involved in making this adjustment but I am hoping it is something that can be easily accommodated.

**Comment by mriscoc** [comment]
You don't need to compile the firmware to change the printing volume, go to the Advanced/Physical settings menu and change there: x max position, y max position, z max position, x bed size and y bed size.

**Reply by TheShaddii** [reply]
That's really all I need to do? Thanks for the info... can't wait to get my machine running good firmware.

**Reply by TheShaddii** [reply]
Actually, I just noticed the Plus Has a touch screen and the regular s1 uses the Knob... is that going to be an issue? I feel like it would be an issue but I don't know enough to know better... Thanks again for your help.

**Reply by mriscoc** [reply]
That's bad, The Professional firmware is NOT compatible with that touch screen.

**Reply by TheShaddii** [reply]
That makes me sad... The default firmware is just terrible...i am going to see how much is gonnna cost to by a knob screen...

**Reply by egonwinter** [reply]
I have the Ender 3 S1 Pro, I buy the Display from the Ender 3/V2 with the Knop and install that brilliant firmware and it work 100 % the only issue what I have, the Hotend reaches only 260 C° butt I think there will come soon any update for it.

**Reply by mriscoc** [reply]
You can use the custom G-code 'C104 U1 T300' to unlock the maximum temperature (300 - 15 overshoot) of the hotend

**Comment by egonwinter** [comment]
Hi Miguel

thanks for your help, I did like you wrote, now I can set it up to 285 C°, 300C° is not working but for the moment it is ok for me, I like that firmware so much, good Job 👍 

Best regards

Egon

Egon Winter

Wilhelm-Schweizer-Straße 10
56564 Neuwied

Mail: ***@***.***
TEL. +49 2631 4009160
TEL. +49 170 6868491
TEL. +49 1520 1506356
Fax. +49 2631 9013170

**Comment by egonwinter** [comment]
Hi Miguel

I found it out, when I do the G Code C104 U1 T315 it goes to 300 C°, thanks again 👍👌

Best regards 

Egon

Egon Winter

Wilhelm-Schweizer-Straße 10
56564 Neuwied

Mail: ***@***.***
TEL. +49 2631 4009160
TEL. +49 170 6868491
TEL. +49 1520 1506356
Fax. +49 2631 9013170
